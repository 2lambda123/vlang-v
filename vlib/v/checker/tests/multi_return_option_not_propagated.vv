fn abc(n int) ?(int, int) {
	if n == 5 {
		return 5, 10
	}
	return none
}

fn main() {
	a, b := abc(10)
	println(a)
	println(b)
}
