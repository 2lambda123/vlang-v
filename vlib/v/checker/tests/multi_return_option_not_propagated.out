vlib/v/checker/tests/multi_return_option_not_propagated.vv:9:10: error: abc() returns an Option, so it should have either an `or {}` block, or `?` at the end
    7 |
    8 | fn main() {
    9 |     a, b := abc(10)
      |             ~~~~~~~
   10 |     println(a)
   11 |     println(b)